it/common/src/main/java/com/google/cloud/teleport/it/common/AbstractPipelineLauncher.java

```diff
@@ -198,8 +213,15 @@ protected JobState handleJobState(Job job) {
   }
 
   /** Creates a JobInfo object from the provided parameters. */
-  protected LaunchInfo getJobInfo(LaunchConfig options, JobState state, Job job, String runner) {
+  protected LaunchInfo getJobInfo(LaunchConfig options, JobState state, Job job) {
     Map<String, String> labels = job.getLabels();
+    String runner = "Dataflow Legacy Runner";
+    Environment environment = job.getEnvironment();
+    if (environment != null
+        && environment.getExperiments() != null
+        && environment.getExperiments().contains("use_runner_v2")) {
```

Comment:
We've used

```
options.addEnvironment("additionalExperiments", Collections.singletonList("use_runner_v2"));
```

To mark use_runner_v2 in some jobs. Not sure if this would've catch it

Reply:
Expermients checked here are queried from dataflow `getJobs` API, which contain `use_runner_v2` even when it is passed from `additionalExperiments` (manually checked to make sure)